File: php/1132.php
<?php

$x = intval(fgets(STDIN));
$y = intval(fgets(STDIN));

if ($x>$y){
    $aux = $x;
    $x = $y;
    $y = $aux;
}

$cont=0;
for ( $i=$x;$i<=$y;$i++){
    if ($i%13!=0){
        $cont+=$i;
    }
}
printf("%d\n", $cont);
?>

File: c/PROBLEMS/1132.c
#include <stdio.h>


int main(){
    int x, y, aux;

    scanf("%d %d", &x, &y);

    if (x>y){
        aux = x;
        x = y;
        y = aux;
    }
    int cont=0;
    for (int i=x;i<=y;i++){
        if (i%13!=0){
            cont+=i;
        }
    }
    printf("%d\n", cont);
}